docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/github.md

```diff
@@ -2,6 +2,189 @@
 sidebar_label: GitHub
 ---
 
-# The GitHub Webhook Receiver
+# GitHub
+
+## Required Secrets for Github
+
+Our webhook receiver will require a Kubernetes secret. This secret is required to contain a `secret` key in its `stringData`. We will need to provide the value we assigned to the `secret` key to GitHub in a later step so keep it handy.
+
+```yaml
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: my-secret
+      namespace: my-namespace
```

Comment:
Most examples throughout the entirety of the docs use Project / ns `kargo-demo`. I think we should stick with that.